### PR TITLE
feat: redact hide

### PR DESCRIPTION
**Pull request by simboonlong**
## Summary

Mainly to allow hide

## Changes

- add `isValueHidden` prop

### CODE REVIEW

File: src/components/common/RedactableValue.tsx [hunk 1]
```diff
@@ -22,12 +22,14 @@ const IconRedact: FunctionComponent = () => {
 };
 
 export interface RedactValueProps {
-  value?: string | number;
-  onRedactionRequested?: () => void;
-  editable?: boolean;
+  value?: string | number | Record<string, unknown> | Record<string, unknown>[];
```
Comment (yapyuyou) [inline src/components/common/RedactableValue.tsx:25]: what are the new record types for?

Comment (simboonlong) [reply src/components/common/RedactableValue.tsx:25]: for obfuscating object or array directly, for [this](https://www.openattestation.com/docs/docs-section/how-does-it-work/document-integrity/#data-obfuscation-limitations)

Comment (yapyuyou) [reply src/components/common/RedactableValue.tsx:25]: oh but in getMessage() we aren't iterating through it if it is an array or obj, what is intended to be displayed?

https://github.com/Open-Attestation/decentralized-renderer-react-components/pull/115/files/97a930fbea782ac64e766020394db563775c299e#diff-c71cdae13f6a78e2030f52df621069b52b6d1542609978fe578e191555267bfaR69

Comment (simboonlong) [reply src/components/common/RedactableValue.tsx:25]: yup, the intention is to use `isValueHidden` when there's a need to obfus array or object. then we can position this UI anywhere we want in the renderer template itself as it deems fit according to the visual design

**Review by yapyuyou — APPROVED** [review]

**Pull request merged**